cfme/tests/containers/test_containers_default_views.py

```diff
@@ -46,6 +52,7 @@ def test_containers_providers_default_view(button_group, view):
               to Grid/Tile/List view
             * Goes to Compute --> Containers --> Providers and verifies the selected view
         """
+    view = sample(VIEWS, 1)
```

Comment:
at first glance this is a intense randomization of the collected test set

is there a good reason not to collect all of them?